### ISSUE TITLE
Error listing zones

### ISSUE THREAD

**Issue by alcidesmig** [googleapis/python-compute#24]
#### Environment details

  - OS type and version: 
  - Python version: Python 3.8.7
  - pip version: 21.0.1 
  - `google-cloud-compute` version:  0.2.0

#### Steps to reproduce

  1. Call list in zones client

#### Code example

```python
from google.cloud import compute
client_zones = compute.ZonesClient()
client_zones.list(project='my-project') #error
```

#### Stack trace
```
Traceback (most recent call last):
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 524, in _ConvertFieldValuePair
    raise ParseError(
google.protobuf.json_format.ParseError: Message type "google.cloud.compute.v1.Zone" has no field named "supportsPzs".
 Available Fields(except extensions): ['availableCpuPlatforms', 'creationTimestamp', 'deprecated', 'description', 'id', 'kind', 'name', 'region', 'selfLink', 'status']

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 574, in _ConvertFieldValuePair
    self.ConvertMessage(item, sub_message)
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 485, in ConvertMessage
    self._ConvertFieldValuePair(value, message)
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 599, in _ConvertFieldValuePair
    raise ParseError(str(e))
google.protobuf.json_format.ParseError: Message type "google.cloud.compute.v1.Zone" has no field named "supportsPzs".
 Available Fields(except extensions): ['availableCpuPlatforms', 'creationTimestamp', 'deprecated', 'description', 'id', 'kind', 'name', 'region', 'selfLink', 'status']

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main2.py", line 20, in <module>
    data = client_zones.list(project='magalu-cliente')
  File "mypath/venv/lib/python3.8/site-packages/google/cloud/compute_v1/services/zones/client.py", line 470, in list
    response = rpc(request, retry=retry, timeout=timeout, metadata=metadata,)
  File "mypath/venv/lib/python3.8/site-packages/google/api_core/gapic_v1/method.py", line 145, in __call__
    return wrapped_func(*args, **kwargs)
  File "mypath/venv/lib/python3.8/site-packages/google/api_core/grpc_helpers.py", line 73, in error_remapped_callable
    return callable_(*args, **kwargs)
  File "mypath/venv/lib/python3.8/site-packages/google/cloud/compute_v1/services/zones/transports/rest.py", line 203, in list
    return compute.ZoneList.from_json(response.content)
  File "mypath/venv/lib/python3.8/site-packages/proto/message.py", line 369, in from_json
    Parse(payload, instance._pb, ignore_unknown_fields=False) #checkpoint_alcides
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 434, in Parse
    return ParseDict(js, message, ignore_unknown_fields, descriptor_pool)
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 454, in ParseDict
    parser.ConvertMessage(js_dict, message)
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 485, in ConvertMessage
    self._ConvertFieldValuePair(value, message)
  File "mypath/venv/lib/python3.8/site-packages/google/protobuf/json_format.py", line 597, in _ConvertFieldValuePair
    raise ParseError('Failed to parse {0} field: {1}.'.format(name, e))
google.protobuf.json_format.ParseError: Failed to parse items field: Message type "google.cloud.compute.v1.Zone" has no field named "supportsPzs".
 Available Fields(except extensions): ['availableCpuPlatforms', 'creationTimestamp', 'deprecated', 'description', 'id', 'kind', 'name', 'region', 'selfLink', 'status'].
```

#### Quick fix (maybe temporary)

Changing `google/cloud/compute_v1/services/zones/transports/rest.py`, line 203 from
```python
        return compute.ZoneList.from_json(response.content)
```
to
```python
        return compute.ZoneList.from_json(response.content, ignore_unknown_fields=True)
```

**Comment by busunkim96** [comment]
Hi @alcidesmig,

Thanks for filing an issue. It looks like this was addressed in the generator templates at https://github.com/googleapis/gapic-generator-python/commit/a70b0787f7e3d40642a4f68574f0cc493cc4e054. I've opened https://github.com/googleapis/googleapis-discovery/pull/23 to bump the generator version.

**Comment by busunkim96** [comment]
Fixed by #25, will be released in the upcoming release 0.2.1. #26
